src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationWithInMethodCallWithSameIndent.java

```diff
@@ -0,0 +1,19 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+/**
+ * Config: default
+ */
+public class InputCommentsIndentationWithInMethodCallWithSameIndent {
+    String s1 = "ONE", s2 = "TWO", s3 = "THREE";
+    private final boolean myList = someMethod(
+        // Some comment here
+        s1,
+        s2,
+        s3
+        // ok
+    );
+
+    private boolean someMethod(String s1, String s2, String s3) {
+        return s1.equals(s2) && s1.equals(s3);
+    }
```

Comment:
to make input shorter, you can make only 2 arguments and change `someMethod` to simple equals

Reply:
Done.

Reply:
I meant
```
    private final boolean myList = equals(
        // Some comment here
        s1,
        s2
        // ok
    );
```

Reply:
Done, I thought you meant change the return of `someMethod` to simple equals.